gameplay/src/Quaternion.cpp

```diff
@@ -219,6 +236,11 @@ void Quaternion::set(const Quaternion& q)
     this->w = q.w;
 }
 
+void Quaternion::set(const Vector3& euler)
```

Comment:
Why? This operation has no axis restriction and is even more common and straightforward than force the user to work with a matrix to only then set the quaternion.